rc/core/make.kak

```diff
@@ -1,4 +1,6 @@
 decl str makecmd make
+decl str make_error_pattern " (?:fatal )?error:"
```

Comment:
I'd let that as a default for now, as it was the previous behaviour, we can discuss later on if the default should be changed.